readme.md

```diff
@@ -53,7 +53,71 @@ Directory names should be lowercase. Filenames should be singular.
 *todo: evaluate common prefix like `pagelib` and BEM naming.*
 
 ## Development setup and workflow
-*Coming soon.*
+The Android and iOS Wikipedia apps make use of the `wikimedia-page-library`. At some point their respective build phases invoke `npm install` which causes a *published* version of this library to be retrieved and used in their respective builds. 
```

Comment:
Thanks for sharing your workflow. I actually find arbitrary word wraps impair code readability whether the line was intended for 80 characters or 500 since code is often all about indentation. I use 80 characters since I've only ever met one person that said that was too large and it works for everyone that can meet that minimum (which is everyone but that one guy) whereas going too wide puts a word wrap requirement on everyone with smaller displays.

Since we have two different approaches, maybe we can compromise? Would 100 characters for code and unlimited for markdown / non-code work for you?